server/controllers/chartsController.js

```diff
@@ -7,18 +7,19 @@ const postProcessData = (data) => {
 
   Object.entries(data).forEach((entry) => {
     const [key, count] = entry
-    const [study, label, color] = key.split('-')
+    const [study, valueLabel, color, studyTarget] = key.split('-')
     const newEntry = {
       color,
       count,
-      label,
+      valueLabel,
```

Comment:
do we need to change anything on the front-end with this change?

Reply:
The label key is unused by us on the front end